gc/base/MemoryPoolSplitAddressOrderedListBase.hpp

```diff
@@ -261,6 +261,7 @@ class MM_MemoryPoolSplitAddressOrderedListBase : public MM_MemoryPoolAddressOrde
 protected:
 	/* Basic free list support */
 	uintptr_t _heapFreeListCount;
+	uintptr_t _maximumHeapFreeListCount; /**< Optimize free lists up to 256 Threads. */
```

Comment:
the comment should explain what this field is used for. for example
maximum value for _heapFreeListCount that can be reinitialized to during VM restore

existing comment is more appropriate at the line the field is initialized, in constructor